draft-ietf-tls-dtls13.md

```diff
@@ -324,7 +324,7 @@ meaning of the fields is unchanged from previous TLS / DTLS versions.
     struct {
         ContentType type;
         ProtocolVersion legacy_record_version;
-        uint16 epoch = 0
```

Comment:
Can we actually change the DTLSPlaintext format like this (recalling that it is used for the initial ClientHello)?

I would mostly have expected to leave DTLSPlaintext unchanged since you are only going to be using it with epoch values less than four.

Reply:
> Can we actually change the DTLSPlaintext format like this (recalling that it is used for the initial ClientHello)?

@kaduk can you elaborate on the problem here? I'm not seeing the issue.

Reply:
Can't we just keep this as uint16 and just call it epoch % 2^16?

Reply:
That seems like the simplest solution here, yeah. We can just track epoch as a uint64 internally, but only take the LSB in the plaintext here.

Reply:
(@ekr reminded me offline that this is a shared structure with DTLS 1.2, so not changing it is best.)

Reply:
> (@ekr reminded me offline that this is a shared structure with DTLS 1.2, so not changing it is best.)

Right, that is the problem I was thinking of.

I think `epoch % 2^16` should work, especially since we don't actually use this structure with large epochs (we've moved to DTLSCiphertext by then).